docs/source/concepts/calibration/TubeCalibrationCorelli.rst

```diff
@@ -0,0 +1,116 @@
+.. _Tube_Calibration_Corelli:
+
+Introduction
+============
+
+Mantid has specific functionality for the Corelli instrument at Oak Ridge National Laboratory (ORNL). The Python scripts are located at: **MANTIDINSTALL**/scripts/corelli/Calibration (`github link <https://github.com/mantidproject/mantid/tree/master/scripts/corelli/calibration>`__).
+These scripts can be launched on MantidWorkbench.
+
+The overarching goal is to create and update the information from the applied calibration functions as new cycles are performed. The result is a database of calibrated vertical "Y"-coordinates stored for each bank as a `TableWorkspace` with pixel_id and Y-coordinate columns.
+
+Performing a Calibration
+========================
+
+The following example illustrates the steps to follow for bank calibration.
+- Load counts from instrument file
+- Select banks to be calibrated
+- Apply the calibration algorithm from :code:`corelli.calibration.utils` 
+
+.. code-block:: Python 
+
+   from corelli.calibration.bank import calibrate_banks
+   from corelli.calibration.utils import apply_calibration, load_banks
+   counts_file = '/SNS/CORELLI/shared/tmp/CORELLI_124023_counts.nxs'
+   load_banks(counts_file, bank_selection='10-19', output_workspace='counts')
+   calibrate_banks('counts', bank_selection='10-19')
+   apply_calibration('counts', 'calib10', output_workspace='counts_10')
+   
+Open the instrument view for workspaces counts and counts_10, then look at bank10 and verify that all tubes in this bank have been calibrated for counts_10
+
+.. image:: ../../images/calibration_corelli.png
+         :alt: center
+         :width: 300px
+
+
+Calibration Database
+====================
+
+Calibrated banks can then be stored in a database which follows the current structure:
+
+.. code-block:: bash
+
+    database_path/
+                  manifest_corelli_20201201.csv 
+                  manifest_corelli_20210601.csv
+                  bank001/
+                          calibration_corelli_bank001_20201201.nxs.h5
+                          mask_corelli_bank001_20201201.nxs.h5
+                          fit_corelli_bank001_20201201.nxs.h5
+                          calibration_corelli_bank001_20210601.nxs.h5
+                          mask_corelli_bank001_20210601.nxs.h5
+                          fit_corelli_bank001_20210601.nxs.h5
+                          
+                          
+                  bank002/
+                  bank003/
+                   ...
+
+where each bank subdirectory contains 3 binary NeXus hdf5 files per calibration cycle (identified by the date timestamp) corresponding to a workspace that be loaded directly on MantidWorkbench for:
```

Comment:
The first sentence is in bold for some reason. Again the bulletted list does not render correctly.

Reply:
for some reason it was adding bold characters without a space after `:`